luigi_pipeline/seqr_loading.py

```diff
@@ -11,12 +11,38 @@
 from lib.model.seqr_mt_schema import SeqrVariantSchema, SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
 
 logger = logging.getLogger(__name__)
-
+GRCh37_STANDARD_CONTIGS = {'1','10','11','12','13','14','15','16','17','18','19','2','20','21','22','3','4','5','6','7','8','9','X','Y', 'MT'}
+GRCh38_STANDARD_CONTIGS = {'chr1','chr10','chr11','chr12','chr13','chr14','chr15','chr16','chr17','chr18','chr19','chr2','chr20','chr21','chr22','chr3','chr4','chr5','chr6','chr7','chr8','chr9','chrX','chrY', 'chrM'}
+OPTIONAL_CHROMOSOMES = ['MT', 'chrM', 'Y', 'chrY']
+VARIANT_THRESHOLD = 100
 
 def check_if_path_exists(path, label=""):
     if (path.startswith("gs://") and not hl.hadoop_exists(path)) or (not path.startswith("gs://") and not os.path.exists(path)):
         raise ValueError(f"{label} path not found: {path}")
 
+def contig_check(mt, standard_contigs, threshold):
+    check_result_dict = {}
+    
+    # check chromosomes that are not in the VCF  
+    row_dict = mt.aggregate_rows(hl.agg.counter(mt.locus.contig))
+    contigs_set = set(row_dict.keys())
+    
+    all_missing_contigs = standard_contigs - contigs_set
+    missing_contigs_without_optional = [contig for contig in all_missing_contigs if contig not in OPTIONAL_CHROMOSOMES]
+
+    if missing_contigs_without_optional:
+        check_result_dict['Missing contig(s)'] = missing_contigs_without_optional
+        logger.warning('Missing the following chromosomes(s):{}'.format(', '.join(missing_contigs_without_optional)))
+                       
+    for k,v in row_dict.items():
+        if k not in standard_contigs:
+            check_result_dict.setdefault('Unexpected string(s)',[]).append(k)
+            logger.warning('Chromosome %s is unexpected.', k)
+        if (k not in OPTIONAL_CHROMOSOMES) and (v < threshold):
```

Comment:
probably should be an elif` instead of an `if`, as we don't really care whether the non-standard chromosome that we don't want in our dataset at all hits the threshhold